app/pages/home-not-logged-in/research.jsx

```diff
@@ -43,7 +39,7 @@ counterpart.registerTranslations('en', {
 const HomePageResearch = (({ count, screenWidth, showDialog, volunteerCount }) =>
   <section className="home-research">
     <Translate className="tertiary-kicker" component="h2" content="researchHomePage.works" />
-    <span className="class-counter">{count.toLocaleString()}</span>
+    <span className="class-counter">{count.toString().replace(/.(?=(?:.{3})+$)/g, '$&,')}</span>
```

Comment:
I was unable to recreate this issue on Safari (browsers not inserting commas in large numbers with `.toLocaleString()`, but this regex should work just the same.